doc/utils/sphinx-config/_themes/lammps_theme/layout.html

```diff
@@ -172,6 +173,11 @@
         </div>
         {%- endblock %}
       </div>
+      <div style="padding: 4px;">
```

Comment:
This will get hidden by the fixed Versions bar.

Reply:
You can do this and get rid of the image.
```html
<div style="position: fixed; bottom: 0px; width: 300px; height: 64px; padding: 4px; background-color: #272525; text-align: center">
<a href="#sidebar-top-target" class="fa fa-caret-up" style="color: white"> Go to top</a>
</div>
```
Note, this still breaks rendering on mobile devices. For that we would have to put this into proper CSS rules that are conditioned on the device size.

Reply:
Ok. I have replaced it. Any idea why I need to click twice on this to have the desired effect?